test/test_core.py

```diff
@@ -6179,6 +6180,20 @@ def test_sigalrm(self):
   def test_signals(self):
     self.do_core_test(test_file('test_signals.c'))
 
+  @parameterized({
+    'sigint': (['sigint', signal.SIGINT, 128 + signal.SIGINT, True]),
+    'sigabrt': (['sigabrt', signal.SIGABRT, 7, False])
+  })
+  def test_sig_default(self, suffix, signal, exit_code, assert_identical):
+    self.do_run_from_file(
```

Comment:
You can use `self.do_core_test` here to avoid specifying the `.out` file.  You can use `out_suffix=suffix` instead.

(You will need to move the .c and .out files into the `test/core` directory too)